docs/docs/user/references/config.md

```diff
@@ -1 +1,15 @@
 # Configuration
+
+KPOps reads most of a [pipeline](/user/getting-started/quick-start/#deploying-the-word-count-pipeline)'s configuration that is unrelated to its [components](./components) from [`config.yaml`](/user/references/config).
+
+Consider enabling [KPOps' editor integration](/user/references/editor-integration) feature to enjoy the benefits of autocompletion and validation when configuring your pipeline.
+
+To learn about any of the available settings, take a look at the example below.
+
+??? "`config.yaml`"
```

Comment:
didn't you have ???+ before to have the panel open automatically?

Reply:
It was on the Editor Integration page. Here I dislike how it looks because the file is too long to fit on the screen in its entirety. Do you think I should expand it by default?

Reply:
ok, makes sende, then it's fine as it is.